internal/cli/tenants.go

```diff
@@ -166,7 +167,13 @@ func addTenantCmd(cli *cli) *cobra.Command {
 			if len(args) == 0 {
 				err := tenantDomain.Pick(cmd, &inputs.Domain, cli.tenantPickerOptions)
 				if err != nil {
-					return err
+					if !errors.Is(err, errUnauthenticated) {
+						return err
+					}
+
+					if err := tenantDomain.Ask(cmd, &inputs.Domain); err != nil {
+						return err
+					}
```

Comment:
We revert to a basic prompt since there's no "list" of tenants yet to formulate / present.